calcom/cal.com
# Make switch thumb color white when disabled.

## What does this PR do?

This will change turned off switch's (thumb) color from black to white.

Fixes # ([issue](https://github.com/calcom/cal.com/issues/8161))


https://www.loom.com/share/cec1d3a85b004f6883e41daf8bd771e8
**Environment**: Staging(main branch) / Production
Main branch.

## Type of change

- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



Open event-types. Turn of show on profile for some meeting
Make sure toggle button does not contain black color in anywhere. 

## Checklist


- I haven't checked if new and existing unit tests pass locally with my changes
https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md does not contain any info about unit tests.
